docs/_snippets/storybook-auto-docs-mdx-docs-dos-donts.md

```diff
@@ -1,12 +1,16 @@
-```mdx filename="Guideline.mdx" renderer="common" language="mdx"
+```jsx filename="Guideline.mdx" renderer="common" language="mdx"
 <Guidelines>
   <Dos>
+    {`
```

Comment:
This shows up in the rendered snippet, which we want to avoid.

I'm not sure why it was necessary to switch `mdx` snippets to `jsx` (maybe prettier wasn't doing anything to `mdx`?), but I don't think it works for our needs.

Reply:
Based on a conversation with @yannbf  It's a known Prettier [issue](https://github.com/prettier/prettier/issues/3837). For now, I'm reverting the changes and ignoring the mdx blocks